external_search.py

```diff
@@ -1656,9 +1709,83 @@ def build(self, _chain_filters=None):
             range_query = self._match_range(
                 'quality', 'gte', self.minimum_featured_quality
             )
-            f = chain(f, range_query)
+            f = chain(f, F('bool', must=range_query))
```

Comment:
When `range_query` is a dictionary and not a real `Filter`, chaining it with the universal base caused a problem. Rather than try to make `_match_range` return a real `Filter` (which seems really tough), I create a boolean `Filter` that just has one argument in `must`.